examples/rsmtool/config_rsmtool.json

```diff
@@ -11,5 +11,6 @@
     "trim_max": 6,
     "id_column": "ID",
     "second_human_score_column": "score2",
-    "length_column": "LENGTH"
+    "length_column": "LENGTH",
+    "standardize_features": false
 }
```

Comment:
I assume you used it for tests? should we keep it or not?

Reply:
Good point! I missed it.

Reply:
@damien2012eng I didn't see this so you can push a fix.